components/LinkGroup/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,202 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+
+import "../index.scss";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS/Navigation/Link Group"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Link Group" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <div class="denhaag-link-group">
+      <h3 class="utrecht-heading-4 denhaag-link-group__caption">Caption</h3>
+      <nav class="denhaag-link-group__navigation" aria-labelledby="denhaag-link-group-navigation-1">
+        <ul
+          id="denhaag-link-group-navigation-1"
```

Comment:
Not sure if this needs an ID?

Reply:
@paulAcato Its for screen readers, see also MDN: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#labeling_section_content

Reply:
@robbieatacato check, looked like a bit of overkill to me ;-)